build.gradle

```diff
@@ -18,7 +18,7 @@ targetCompatibility = 1.8
 
 group = "com.unstoppabledomains.resolution"
 archivesBaseName = "resolution"
-version = "1.0.9"
+version = "1.1.0"
```

Comment:
I just double-checked v1.1.0 is already published. Let's update it to 1.2.0

Reply:
Ok, I see now, we have two different artifacts at this point: https://search.maven.org/search?q=com.unstoppabledomains
Bumped to v1.2.0 now, thanks!